src/components/NotificationsDrawer/DrawerPanelContent.tsx

```diff
@@ -114,14 +122,29 @@ const DrawerPanelBase = ({ innerRef }: DrawerPanelProps) => {
     toggleDrawer(false);
   };
 
-  const onMarkAllAsRead = () => {
-    updateNotificationsStatus(true);
-    setIsDropdownOpen(false);
+  const onUpdateSelectedStatus = (read: boolean) => {
+    axios
+      .put('/api/notifications/v1/notifications/drawer/read', {
+        notification_ids: selectedNotifications.map((notification) => notification.id),
+        read_status: read,
+      })
+      .then(() => {
+        selectedNotifications.forEach((notification) => updateNotificationRead(notification.id, read));
+        setIsDropdownOpen(false);
+        updateAllNotificationsSelected(false);
```

Comment:
Is this supposed to be always false?

Reply:
I'm doing this so that the dropdown closes and the notifications are deselected after the selected notifications are marked read/unread. 

If we would like to keep those notifications selected and/or the dropdown open, then I can update this.

Reply:
Nono this is good. I am trying to be extra careful when I see literal values :slightly_smiling_face: